src/index.html

```diff
@@ -15,18 +15,38 @@
       rel="stylesheet"
       href="style.css"
     />
+    <link
+      rel="stylesheet"
+      href="fonts/Avenir-Book.ttf"
+    />
+    <link
+      rel="stylesheet"
+      href="fonts/Avenir-Heavy.ttf"
+    />
   </head>
   <body>
```

Comment:
Add an empty line between sibling elements to improve readability. There should be an empty line between the `head` and the `body` tags.